edge-agent/docker/linux/arm64v8/Dockerfile

```diff
@@ -1,6 +1,26 @@
-ARG base_tag=1.0.6.15-linux-arm64v8
+ARG base_tag=3.1.19-bionic-arm64v8
+ARG num_procs=4
```

Comment:
why was this added?

Reply:
@nimanch It was in the original base image: See https://github.com/Azure/iotedge/blob/release/1.2/edge-agent/docker/linux/arm64v8/base/Dockerfile

...but it doesn't need to be there.